tools/deb/BUILD

```diff
@@ -0,0 +1,51 @@
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
+
+pkg_tar(
+    name = "envoy-bin",
+    srcs = [
+        "istio-iptables.sh",
+        "istio-start.sh",
+        "//src/envoy/mixer:envoy",
+    ],
+    mode = "0755",
+    package_dir = "/usr/local/bin",
```

Comment:
AFAIK that's for distribution-provided binaries ( i.e. if debian or ubuntu would build the package). 
Google packages typically go to /opt/ (chrome), that would be my first choice.

However - for now I think it's important to minimize disruption, the docker images use /usr/local/bin and it's hardcoded into pilot agent - no need to confuse users with 2 locations. After we decide where it should go - we can update agent, docker, etc and keep things consistent.